pkg/scheduler/framework/plugins/nodevolumelimits/csi.go

```diff
@@ -82,7 +82,7 @@ func (pl *CSILimits) Filter(ctx context.Context, _ *framework.CycleState, pod *v
 	csiNode, err := pl.csiNodeLister.Get(node.Name)
 	if err != nil {
 		// TODO: return the error once CSINode is created by default (2 releases)
-		klog.V(5).Infof("Could not get a CSINode object for the node: %v", err)
+		klog.V(5).InfoS("Could not get a CSINode object for the node", "err", err)
```

Comment:
https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/migration-to-structured-logging.md#change-log-functions-to-structured-equivalent

Maybe is `klog.ErrorS(err, "Could not get a CSINode object for the node")`

Reply:
`klog.V(5).ErrorS(err, "Could not get a CSINode object for the node")` is ok ?